boot/bootutil/src/bootutil_public.c

```diff
@@ -262,6 +264,7 @@ boot_read_swap_state(const struct flash_area *fap,
     off = boot_swap_info_off(fap);
     rc = flash_area_read(fap, off, &swap_info, sizeof swap_info);
     if (rc < 0) {
+        BOOT_LOG_DBG("boot_read_swap_state: error %d reading state", rc);
```

Comment:
For all places where code exits with BOOT_EFLASH i would go with BOOT_LOG_ERR()

Reply:
I have removed the logging here. At this point I am trying to avoid logging errors on read/write operations; they increase the code size and actually do nothing when flash is breaking, which is most probable case for these failures.